common/cultures/culture_ancient.txt

```diff
@@ -150,6 +150,48 @@ ancient_group = {
 		mat_grm_name_chance = 40
 		mother_name_chance = 5
 		
+		# Modifiers
+		modifier = default_culture_modifier
+	}
+	ancient_coral = {
```

Comment:
You should add cultural building and retinue for their culture.
Here are the docs for building balance.
https://docs.google.com/spreadsheets/d/1wLs_i8YQRL4_BmxPBdoibJQTC0vJmu8RbidPy7cwGZY/edit#gid=2044635640
Retinues are balanced around having the same sum of bonuses.

Reply:
they already have building from their culture group. added retinue